mscore/magbox.cpp

```diff
@@ -67,7 +67,7 @@ MagBox::MagBox(QWidget* parent)
       setToolTip(tr("Zoom"));
       setWhatsThis(tr("Zoom"));
       setValidator(new MagValidator(this));
-      setAutoCompletion(false);
+      setCompleter(0);
```

Comment:
Maybe `nullptr` would be better with our code style

Reply:
I can sure change that.

Reply:
But the documentations says:
Sets the completer to use instead of the current completer. If completer is 0, auto completion is disabled.

Reply:
`setCompleter` takes a pointer so `0` would be the same as `nullptr` here. This is just a matter of code style.

Reply:
Dou you still want me to change this?

Reply:
Doesn't matter much for me, if something else is also changed it might be good to do. But we can surely leave it as it is.

Reply:
done